libs/stocks/feature-shell/src/lib/stocks/stocks.component.ts

```diff
@@ -34,10 +35,11 @@ export class StocksComponent implements OnInit {
 
   ngOnInit() {}
 
-  fetchQuote() {
+  fetchQuote = () => {
```

Comment:
changed it to lambda function otherwise line 40 "this.stockPickerForm.valid" was giving error because this was not instance of class without lambda function